Headers/AppKit/AppKit.h

```diff
@@ -141,6 +141,7 @@
 #import <AppKit/NSDocumentController.h>
 #import <AppKit/NSDrawer.h>
 #import <AppKit/NSFileWrapper.h>
```

Comment:
Why do you still import the old, now empty file AppKit/NSFileWrapper.h?
The same is true in a few places further down in other files.

Reply:
Because it's not empty.  It imports the one in foundation.   This is what is done on macOS.

Reply:
In the next line you are importing the new extension file, which also includes the Foundation version of NSFileWrapper. So this additional line is not needed.